src/components/History/Timeline/style.scss

```diff
@@ -0,0 +1,26 @@
+#timeline {
+	padding-bottom: 20px;
```

Comment:
instead of adding padding to timeline, can you add 40px of margin onto `.history-page .content-section`? That's how it's done for the about and sponsor pages (although I noticed there was some minor errors with our codebase here, so I'll do some refactoring later). Look at `/about/styles` for an example of what i mean.